### ISSUE TITLE
Updating Migration Guide to be more digestable

### ISSUE THREAD

**Issue by DanielDornhardt** [meteor/meteor#12818]
Hi,

When checking the migration guide, I noticed that there are a lot of linked ressources, but no overall "game plan" for users so far.

I'd like to reorganize it a bit & put it more into a guide format which we then keep up-to-date as things develop towards 3.0.

@Grubba27 is that cool with you?

I'll work on this and create a WIP PR when I have something.

**Comment by 07tAnYa** [comment]
can I work on this?

**Comment by StorytellerCZ** [comment]
@07tAnYa feel free to submit PR.

**Comment by 07tAnYa** [comment]
> @07tAnYa feel free to submit PR.

Sir , I have to discuss something related to this? Is this the right place for discussion?

**Comment by StorytellerCZ** [comment]
@07tAnYa yes, that is what issues are for. This is probably going to have a lot of discussion. Some can be resolved here, other things will most likely be talked about once the PR is in, but since it is guide most of it can probably be resolved here. Also note that @DanielDornhardt has some specific ideas.

**Comment by 07tAnYa** [comment]
> @07tAnYa yes, that is what issues are for. This is probably going to have a lot of discussion. Some can be resolved here, other things will most likely be talked about once the PR is in, but since it is guide most of it can probably be resolved here. Also note that @DanielDornhardt has some specific ideas.

ok sir

**Comment by 07tAnYa** [comment]
@DanielDornhardt  Sir, in your work how may I help you I also have some ideas for the guide. Should we collaborate on a g-meet or any preferred platform by you?

**Comment by Grubba27** [comment]
The migration docs were rewritten as the dedicated Meteor 3 migration guide site when 3.0 shipped, which fulfills what this issue proposed. Closing.
